dynamixel_workbench_single_manager/src/dynamixel_driver.cpp

```diff
@@ -982,12 +993,14 @@ int main(int argc,char** argv)
   double publish_rate; // hz
   int baud_rate;
   std::string device_id;
+  std::string joint;
   ph.param("publish_rate",publish_rate,50.0);
   ph.param("baud_rate",baud_rate,2000000);
   ph.param("device_id",device_id,std::string("/dev/ttyUSB0"));
+  ph.param("joint", joint, std::string(""));
```

Comment:
I think having an empty string could make it harder to find bugs related to forgetting to set the joint ros parameter, plus there's no harm  in having a default name such as "dynamixel_joint" or something along those lines.